views/js/qtiCreator/editor/styleEditor/fontSizeChanger.js

```diff
@@ -99,25 +107,23 @@ define(['jquery', 'lodash', 'taoQtiItem/qtiCreator/editor/styleEditor/styleEdito
          */
         $resetBtn.on('click', function () {
             $input.val('');
-            styleEditor.apply(itemSelector, 'font-size');
-            styleEditor.apply(figcaptionSelector, 'font-size');
+            styleEditorApply(null);
             itemFontSize = parseInt($item.css('font-size'), 10);
         });
 
         /**
          * style loaded from style sheet
          */
-        const oldSelector = `${itemSelector} *`;
```

Comment:
Was the `oldSelector` removed because we don't need it any more?

Reply:
Yes, sorry, forgot to comment on that.
"New" selector was added 4 years ago https://github.com/oat-sa/extension-tao-itemqti/commit/1f1634311687959ed221d3d5e8417f02e165af82 , so I thought we can remove the "old" one now.